frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/review-and-create-step/review-and-create-step.tsx

```diff
@@ -1,224 +1,128 @@
 import * as React from 'react';
 import { useTranslation } from 'react-i18next';
-import {
-  TextContent,
-  Text,
-  TextVariants,
-  Card,
-  CardBody,
-  AlertVariant,
-  Split,
-  SplitItem,
-} from '@patternfly/react-core';
-import {
-  GreenCheckCircleIcon,
-  YellowExclamationTriangleIcon,
-  BlueInfoCircleIcon,
-  RedExclamationCircleIcon,
-  useFlag,
-} from '@console/shared/src';
+import { TextContent, Text, TextVariants, List, ListItem } from '@patternfly/react-core';
+import { useFlag } from '@console/shared/src';
 import { humanizeBinaryBytes } from '@console/internal/components/utils';
-import { MINIMUM_NODES, NetworkTypeLabels, NO_PROVISIONER } from '../../../../constants';
+import { BackingStorageType } from '../../../../constants/create-storage-system';
+import { NetworkTypeLabels, NO_PROVISIONER } from '../../../../constants';
 import {
-  capacityAndNodesValidate,
   getAllZone,
+  getExternalStorage,
   getTotalCpu,
   getTotalMemory,
 } from '../../../../utils/create-storage-system';
-import { OSD_CAPACITY_SIZES, TotalCapacityText } from '../../../../utils/osd-size-dropdown';
-import {
-  getEncryptionLevel,
-  ValidationMessage,
-  VALIDATIONS,
-  ValidationType,
-} from '../../../../utils/common-ocs-install-el';
-import { WizardNodeState, WizardState } from '../../reducer';
-import { NetworkType } from '../../../../types';
+import { OSD_CAPACITY_SIZES } from '../../../../utils/osd-size-dropdown';
+import { WizardState } from '../../reducer';
 import { GUARDED_FEATURES } from '../../../../features';
 import './review-and-create-step.scss';
 
-const NodesCard: React.FC<NodeCardProps> = ({ nodes }) =>
-  !!nodes.length && (
-    <Card isCompact isFlat component="div" className=".odf-review-and-create__card">
-      <CardBody isFilled className=".odf-review-and-create__card-body">
-        <TextContent>
-          {nodes.map((node) => (
-            <Text key={node.name} component={TextVariants.small}>
-              {node.name}
-            </Text>
-          ))}
-        </TextContent>
-      </CardBody>
-    </Card>
-  );
-
-type NodeCardProps = { nodes: WizardNodeState[] };
-
-const REVIEW_ICON_MAP = {
-  [AlertVariant.success]: GreenCheckCircleIcon,
-  [AlertVariant.warning]: YellowExclamationTriangleIcon,
-  [AlertVariant.info]: BlueInfoCircleIcon,
-  [AlertVariant.danger]: RedExclamationCircleIcon,
-};
-
-export const ReviewListTitle: React.FC<ReviewListTitleProps> = ({ text }) => (
-  <dt>
+export const ReviewItem = ({ children, title }) => (
+  <div className="odf-review-and-create__review-item">
```

Comment:
odf-review-and-create__review--item I saw above you used double hyphen symbol :)

Reply:
I cant find that occurrence, can you point line no ?

Reply:
https://github.com/openshift/console/pull/9815/files#diff-f3f8fdf9bc309dbd09ce0bcc4b050057d4e80509065e52d4fb110be412821b26R51

Reply:
odf-backing-storage__selection--width

Reply:
so we use it only for a property?

Reply:
I think `review-item` is one word that is why may be ?